### PR TITLE
awesome book: with es6 pull request

### PR DESCRIPTION
**Pull request by theresetuyi**
## awesome book

-Use ES6 modules to write modular JavaScript.
-Learn how to use proper ES6 syntax.
-Understand improvements that ES6 brings to JavaScript.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/* eslint-disable */
```
Comment (Gambit142) [inline index.js]: - [ ] Kindly remove all `Eslint-disable` statements from this file. `Eslint` ensures we write clean and understandable codes and using a disable statement defeats this purpose. Kindly remove all `eslint-disable` statements in the file 👍

File: modules/bookDesplay.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+// eslint-disable-next-line no-unused-vars
```
Comment (Gambit142) [inline modules/bookDesplay.js]: - [ ] Kindly remove all `Eslint-disable` statements from this file. `Eslint` ensures we write clean and understandable codes and using a disable statement defeats this purpose. Kindly remove all `eslint-disable` statements in the file 👍

File: modules/books.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+// eslint-disable-next-line no-unused-vars
```
Comment (Gambit142) [inline modules/books.js]: - [ ] Kindly remove all `Eslint-disable` statements from this file. `Eslint` ensures we write clean and understandable codes and using a disable statement defeats this purpose. Kindly remove all `eslint-disable` statements in the file 👍

File: modules/bookDesplay.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      this.books.push(newBook);
+      this.render();
+      this.saveBooks();
+    }
+
+    removeBook(id) {
+      this.books = this.books.filter((book) => book.id !== id);
+      this.render();
+      this.saveBooks();
+    }
```
Comment (Gambit142) [inline modules/bookDesplay.js]: - [ ] I noticed that I can't add or remove a book. Whenever I try to, I get this error in the console. Kindly ensure that users can add or remove books in this milestone 👍 

image

File: index.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+let display;
+const today = document.getElementById('today');
+
+window.onload = () => {
+  // eslint-disable-next-line no-undef
+  display = new Display();
+  display.render();
+  today.textContent = new Date().toDateString();
+};
+
```
Comment (Gambit142) [inline index.js]: - [ ] According to the requirements, you should display the time in your application by using the `DateTime` object from the `Luxon` library. Kindly implement this feature. 👍

File: modules/bookDesplay.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+        // eslint-disable-next-line no-undef
+        this.books = JSON.parse(localStorage.getItem('books')).map((book) => new Book(book.title, book.author, book.id));
+      }
+    }
+
+    addBook(newBook) {
+      this.books.push(newBook);
+      this.render();
+      this.saveBooks();
+    }
```
Comment (Gambit142) [inline modules/bookDesplay.js]: - [ ] Kindly use `ES6` **_arrow syntax_** for all functions in this class. Remember that the requirements state that all functions should be written in `ES6`. Kindly implement this change. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @theresetuyi 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- Good use of const and let keywords ✔️ 
- `modules` directory is present ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by sumairq — APPROVED** [review]
### Hi @theresetuyi , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**